### ISSUE TITLE
Continuous Commit Push?

### ISSUE THREAD

**Issue by JakeSays** [microsoft/Extensible-Storage-Engine#27]
When will the "Continuous Commit Push" begin?

**Comment by MsftBrettShirley** [comment]
I assume this means, we push code to github for each commit we do on the ESE code base internally?

**Comment by JakeSays** [comment]
Yeah that's basically what I mean. There is a paragraph on it in the README.md.

**Comment by MsftBrettShirley** [comment]
Yeah I am still working on cleaning up the code base ... step 1 from our discussion thread over in the Discussions tab.  The next big push, OR _maybe_ one or two after that will be a good signal that I'm probably onto step 2 / getting continuous push done.
